sw/device/tests/BUILD

```diff
@@ -5318,13 +5325,16 @@ opentitan_test(
         test_cmd = "--elf={sram_program}",
         test_harness = "//sw/host/tests/chip/rv_core_ibex_epmp",
     ),
-    exec_env = {
-        # This test is not compatible with the test_rom because the test_rom
-        # does not respect the exec_disabled OTP config word.
-        "//hw/top_earlgrey:fpga_cw310_sival": None,
-        "//hw/top_earlgrey:fpga_cw310_sival_rom_ext": None,
-        "//hw/top_earlgrey:fpga_cw310_rom_with_fake_keys": None,
-    },
+    exec_env = dicts.add(
+        EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
```

Comment:
Please avoid adding these to rv_core_ibex tests, they don't work on silicon and I'd like to avoid future merge conflicts.

Reply:
Removed